docs/modules/mapstore/pages/configuring-a-generic-maploader.adoc

```diff
@@ -375,6 +375,14 @@ mapConfig.setMapStoreConfig(mapStoreConfig);
 
 |===
 
+== Supported backends
+
+You can use any database as the MapStore backend as long as you have it's Hazelcast SQL Connector on the classpath.
+
+Officially supported backend databases:
+
+- MySQL, PostgreSQL (it uses JDBC SQL Connector).
+- MongoDB (make sure you have `hazelcast-jet-mongodb` artifact included on the classpath).
```

Comment:
This could be made clearer

- GenericMapStore needs SQL Connector which supports SELECT, UPDATE, SINK and DELETE statements.
- JDBC connector needs appropriate driver

Reply:
https://github.com/hazelcast/hz-docs/pull/833/commits/639884dc68d562b82550099eefc2a9cadc8767e7